### DISCUSSION TITLE
css not loaded properly?

### DISCUSSION THREAD

**Discussion by yagnikvadi2003** [webpack/webpack#18706] [Q&A]
bebel:     "presets": ["@babel/preset-env","@babel/preset-react", "@babel/preset-typescript"]

"css-loader": "^7.1.2",
    "sass": "^1.77.8",
    "sass-loader": "^16.0.1",
    "style-loader": "^4.0.0",

{
                test: /\.css$/,
                // include: path.resolve(__dirname, "..", "src"),
                exclude: /node_modules/,
                use: [
                    MiniCssExtractPlugin.loader,
                    {
                        loader: "css-loader",
                        options: {
                            sourceMap: true,
                            esModule: false,
                        },
                    },
                ],
            },
            {
                test: /\.scss$/,
                use: [
                    MiniCssExtractPlugin.loader,
                    'css-loader',
                    'sass-loader',
                ],
            },    
    
            new MiniCssExtractPlugin({
            // filename: '[id].css', // environment !== 'production' ? '[name].[contenthash].css' : '[name].css',
            filename: '[name].[contenthash].css',
            chunkFilename: '[id].[contenthash].css',
            runtime: false,
        }),

![Screenshot from 2024-08-25 00-47-08](https://github.com/user-attachments/assets/c2bef582-784d-4cb3-90d8-15168aa3dc0b)

CSS-loder is not properly working is there your side issues ya my side issues please guide me..!

**Other relevant information:**
webpack version:  5.94.0
Node.js version: 20.17.0
Operating System: ubuntu 22.04
Additional tools:

**Comment by alexander-akait** [comment]
What do you mean?
